.php-cs-fixer.dist.php

```diff
@@ -4,8 +4,8 @@
     ->in(__DIR__)
 ;
 
-return PhpCsFixer\Config::create()
-    ->setRules(array(
+$config = new PhpCsFixer\Config();
+return $config->setRules(array(
         '@Symfony' => true,
```

Comment:
The indentation seems weird here

Reply:
Not too sure what you are expecting here, but I've made a change that hopefully helps.